hack/build/run-functional-tests.sh

```diff
@@ -95,14 +95,10 @@ fi
 
 (
     export TESTS_WORKDIR=${CDI_DIR}/tests
-    ginkgo_args="--trace --timeout=8h --v"
-
-    if [[ -n "$CDI_E2E_SKIP" ]]; then
-        ginkgo_args="${ginkgo_args} --skip=${CDI_E2E_SKIP}"
-    fi
+    ginkgo_args="--trace --timeout=8h --v --skip=${CDI_E2E_SKIP} --label-filter=${CDI_LABEL_FILTER}"
```

Comment:
Are we sure that an empty skip value will not mess things up?

Reply:
I tried it on my side without issues, but decided for this out of pure aesthetic reasons.
On a second thought being explicit may be the better choice

Reply:
TBH I took a look at the test set that was exercised, everything looks the same, so it's fine